### PR TITLE
Client go upgrade v6

### PR DESCRIPTION
**Pull request by alexeyklyukin**
Upgrade go-client to v6, implement deepcopy for CRDs, cope with some minor API changes (and packages moved).

### CODE REVIEW

**Comment by alexeyklyukin** [comment]
👍

File: glide.yaml [hunk 1]
```diff
@@ -10,38 +10,14 @@ import:
...
-  - kubernetes/typed/extensions/v1beta1
-  - pkg/api
-  - pkg/api/v1
-  - pkg/apis/apps/v1beta1
-  - pkg/apis/extensions/v1beta1
-  - rest
-  - tools/cache
-  - tools/clientcmd
-  - tools/remotecommand
+  version: ^6.0.0
```
Comment (zerg-junior) [inline glide.yaml]: what's so specific about version  6 ? afaik current docs [recommend v8](https://github.com/kubernetes/client-go#client-go)

File: pkg/controller/postgresql.go [hunk 2]
```diff
@@ -147,12 +147,12 @@ func (d *crdDecoder) Decode() (action watch.EventType, object runtime.Object, er
 
 func (c *Controller) clusterWatchFunc(options metav1.ListOptions) (watch.Interface, error) {
 	options.Watch = true
+	// MIGRATION: FieldsSelectorParam(nil)
```
Comment (zerg-junior) [inline pkg/controller/postgresql.go]: does it mean that the code line was removed during migration cause the method is  not there anymore ?

**Comment by zerg-junior** [comment]
I successfully run the operator with the updated client locally, so +1

**Comment by zerg-junior** [comment]
:+1:

File: glide.yaml (hunk 1 above)
Comment (alexeyklyukin) [reply glide.yaml]: We want to do one step at a time, upgrade to v6, achieve our goal of implementing priority, then go further with 8.

File: pkg/controller/postgresql.go (hunk 2 above)
Comment (alexeyklyukin) [reply pkg/controller/postgresql.go]: Yes

**Pull request merged**